jax/experimental/jax2tf/jax2tf.py

```diff
@@ -501,7 +501,7 @@ def _population_count(x):
 
 def _div(lhs, rhs):
   if lhs.dtype.is_integer:
-    quotient = tf.math.floor_divide(lhs, rhs)
+    quotient = tf.math.floordiv(lhs, rhs)
```

Comment:
This suggests that we have no tests for _div for integer types?

@SIben was looking at expanding test coverage. FYI.

Reply:
Yes, it is somehow the same problem that we had for `add` and `mul` – it is only included in `test_binary_elementwise` afaict, which only considers floating point types. Perhaps it'd be good to gather the untested paths by dtype to get a better overview.